conftest.py

```diff
@@ -227,6 +227,11 @@ def sr_disk(request, host):
             f"disk or block device {disk} is either not present or already used on master host"
     yield disk
 
+@pytest.fixture(scope="session")
+def sr_disk_devid(host, sr_disk):
```

Comment:
Making this fixture depend on `sr_disk` will make it impossible to run all storage tests on the same host in the same test job.

I believe a 4K disk should depend on a separate fixture for 4K disks, so that a host can have two disks, one with 512 blocks (`sr_disk`) and one with 4K blocks (`sr_4k_disk`). As suggested in https://github.com/xcp-ng/xcp-ng-tests/pull/215 where I got no answer.

Reply:
This PR is independent from the LargeBlock one because this fixture could be used by other tests that need a unique identifier for its disks. I'm gonna address the need for a special fixture for 4KiB disks in the #215 . If you do not want this fixture to be added, this PR can be closed.